docs/history.apib

```diff
@@ -140,30 +144,30 @@ If no query string is set, then all values from all attributes are returned.
                   },
                   {
                     "device_id": "b374a5",
-                    "ts": "2018-02-14T13:41:33.306000Z",
+                    "ts": "2010-01-01T10:13:22.306000Z",
                     "metadata": {
                         "protocol": "mqtt",
                         "payload": "json"
                     },
-                    "value": "24.10",
+                    "value": "22.12",
                     "attr": "temperature"
                   }
                 ],
               "pressure":
                 [
                   {
                     "device_id": "b374a5",
-                    "ts": "2018-02-14T13:43:33.306000Z",
+                    "ts": "2010-01-01T10:11:22.306000Z",
                     "metadata": {
                         "protocol": "mqtt",
                         "payload": "json"
                     },
-                    "value": "1013",
+                    "value": "1053",
                     "attr": "pressure"
                   },
                   {
                     "device_id": "b374a5",
-                    "ts": "2018-02-14T13:42:33.306000Z",
+                    "ts": "2010-01-01T10:12:22.306000Z",
```

Comment:
Por que 2010?

Reply:
The idea was to bring the oldest records, but I just saw that I forgot to change the parameter `dateFrom=2010-01-01T00:00:00Z` as well :sweat_smile: